docs/Project-Configuration/Merge-Queue.md

```diff
@@ -117,4 +117,83 @@ branch protection rule for one or more "evergreen/variant_name" statuses. You
 might wish to do this if you wish more variants to run in a PR than are actually
 required to merge. For example, there may be long-running tasks which users care
 about only some of the time, and you do not wish to block PRs or merges on those
-tasks, but you still wish them to run automatically.
\ No newline at end of file
+tasks, but you still wish them to run automatically.
+
+**Q:** How can I turn off the merge queue to block users from merging?
+
+**A:** Check the "Lock branch" setting in the branch protection rules. There are
+*no changes to make on the Evergreen side.
+
```

Comment:
Can we add a Q&amp;A for whether or not users are able to see patch/patches associated with their merge attempt (and if not, anything else they can look for to diagnose simple problems themselves)?